src/main/java/com.gigssandbox/UserMessenger.java

```diff
@@ -0,0 +1,79 @@
+package com.gigssandbox;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Properties;
+import java.util.Scanner;
+
+class UserMessenger {
+
+    private static final Scanner in = new Scanner(System.in);
+    private Properties properties = new Properties();
+
+    UserMessenger() {
+        try {
+            properties.load(new FileInputStream("src/main/resources/strings.properties"));
+        } catch (IOException e) {
+            sendMessage("properties_error");
+            System.exit(1);
+        }
+    }
+
+    String readString() {
```

Comment:
readString and nextCommand are in two different level of abstraction. readString is too low level